src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -207,24 +237,29 @@ const ChartsHolder = React.memo(({ region, chartId }: ChartsHolderProps) => {
                 />
               </LocationPageBlock>
             )}
-            {ALL_METRICS.map(metric => (
-              <ErrorBoundary key={metric}>
-                {!projections ? (
-                  <LoadingScreen />
-                ) : (
-                  <LocationPageBlock>
-                    <ChartBlock
-                      metric={metric}
-                      projections={projections}
-                      chartRef={metricRefs[metric]}
-                      isMobile={isMobile}
-                      region={region}
-                      stats={stats}
-                    />
-                  </LocationPageBlock>
-                )}
-              </ErrorBoundary>
-            ))}
+            {CHART_GROUPS.map((group: ChartGroup) => {
+              const { groupHeader } = group;
+              const groupRef = chartBlockRefs[groupHeader];
+              return (
+                <ErrorBoundary key={group.groupHeader}>
```

Comment:
nit: Maybe use `groupHeader` directly since it's a variable above? (or remove the variable and just use `group.groupHeader`